### DISCUSSION TITLE
Complet rescan resets the edited meta information

### DISCUSSION THREAD

**Discussion by al13nus** [photoprism/photoprism#2238] [Ideas]
Hey together,

I found out today, that if you edit the "Local Time" of a picture and the start the "Complete Rescan", its gets resetet to what is in the file. In my opinion all the edits should be directly writen to the file back or at least saved in the database, so the rescaning doesnt overwrite it back. Am I'm missing something here?

**Comment by lastzero** [comment]
Generally should not happen as the time gets flagged as manually edited... 

- I assume you used the Web UI and didn't directly update the database table? 
- What exact version / build are you using? 
- Does it also happen on our public demo? 
- If so, could you write down the exact steps to reproduce and open a bug ticket with this information? Debug logs from your personal instance may be helpful as well (demo is not running in debug mode).

**Reply by al13nus** [reply]
Difficult for me to reproduce, because I'm in the process of recreating my librariy (you remember maybe my help call because I lost all my originals haha). If flagging as manually edited was implemented, then I will assume for now that it was my mistake. Maybe I will make some tests later and report back to you, if it still doesn't work.

**Reply by lastzero** [reply]
Just use demo.photoprism.app then 👍

**Reply by lastzero** [reply]
Tested. Preserving the value works when you also change the date along with the time, just not when you change the time alone.

**Reply by lastzero** [reply]
See:
- https://github.com/photoprism/photoprism/issues/2239
